src/Compilers/Core/Portable/Operations/IVariableDeclaration.cs

```diff
@@ -14,13 +14,13 @@ namespace Microsoft.CodeAnalysis.Semantics
     public interface IVariableDeclaration : IOperation
     {
         /// <summary>
-        /// Variable declared by the declaration.
+        /// Symbols declared by the declaration.
         /// </summary>
-        ILocalSymbol Variable { get; }
+        ImmutableArray<ILocalSymbol> Variables { get; }
         /// <summary>
         /// Initializer of the variable.
```

Comment:
perhaps doc that in VB it's possible to create multiple variables at a single time with the same initialized value.